src/Sensors.cpp

```diff
@@ -711,8 +711,15 @@ void Sensors::aht10Init() {
 }
 
 void Sensors::CO2scd30Init() {
-    DEBUG("-->[SCD30] starting SCD30 sensor..");
+    DEBUG("-->[SCD30] starting CO2 SCD30 sensor..");
     scd30.begin();
+    delay(5);
+    CO2scd30Read();
+    if (CO2 > 0) {
+        Serial.println("-->[SCD30] detected!");      
+        device_selected = "SCD30";
+        device_type = 6;
+    }
```

Comment:
For now it's right, but we need refactored all read methods for return boolean, please only change println call to DEBUG method.

Reply:
done